### ISSUE TITLE
Can't "yum update" on Centos

### ISSUE THREAD

**Issue by leodip** [dotnet/core#1773]
When I try to "yum update" on Centos, I get the output below.

(...)
```
---> Package systemtap-sdt-devel.x86_64 0:3.2-8.el7_5 will be an update
--> Running transaction check
---> Package dotnet-runtime-2.1.x86_64 0:2.1.2-1 will be an update
--> Processing Dependency: dotnet-runtime-deps-2.1 >= 2.1.2 for package: dotnet-runtime-2.1-2.1.2-1.x86_64
---> Package python-requests-toolbelt.noarch 0:0.8.0-1.el7 will be installed
--> Finished Dependency Resolution
--> Running transaction check
---> Package dotnet-runtime-2.1.x86_64 0:2.1.2-1 will be an update
--> Processing Dependency: dotnet-runtime-deps-2.1 >= 2.1.2 for package: dotnet-runtime-2.1-2.1.2-1.x86_64
---> Package kernel.x86_64 0:3.10.0-693.17.1.el7 will be erased
--> Finished Dependency Resolution
Error: Package: dotnet-runtime-2.1-2.1.2-1.x86_64 (packages-microsoft-com-prod)
           Requires: dotnet-runtime-deps-2.1 >= 2.1.2
           Installed: dotnet-runtime-deps-2.1-2.1.1-1.x86_64 (@packages-microsoft-com-prod)
               dotnet-runtime-deps-2.1 = 2.1.1-1
           Available: dotnet-runtime-deps-2.1-2.1.0-1.x86_64 (packages-microsoft-com-prod)
               dotnet-runtime-deps-2.1 = 2.1.0-1
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest
[root@li1296-24 ~]#
```

My "dotnet --info" is:

```
[root@li1296-24 ~]# dotnet --info
.NET Core SDK (reflecting any global.json):
 Version:   2.1.301
 Commit:    59524873d6

Runtime Environment:
 OS Name:     centos
 OS Version:  7
 OS Platform: Linux
 RID:         centos.7-x64
 Base Path:   /usr/share/dotnet/sdk/2.1.301/

Host (useful for support):
  Version: 2.1.1
  Commit:  6985b9f684

.NET Core SDKs installed:
  2.1.104 [/usr/share/dotnet/sdk]
  2.1.301 [/usr/share/dotnet/sdk]

.NET Core runtimes installed:
  Microsoft.AspNetCore.All 2.1.1 [/usr/share/dotnet/shared/Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.App 2.1.1 [/usr/share/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 2.0.6 [/usr/share/dotnet/shared/Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.1.1 [/usr/share/dotnet/shared/Microsoft.NETCore.App]

To install additional .NET Core runtimes or SDKs:
  https://aka.ms/dotnet-download
[root@li1296-24 ~]#
[root@li1296-24 ~]#

```

Can anyone help me?

Thanks

**Comment by Scandal-UK** [comment]
They haven't properly released 2.1.2 on YUM. I'm sure they will fix it soon...

https://github.com/dotnet/core/issues/1765

**Comment by Petermarcu** [comment]
Issue moved to [dotnet/core-setup #4365](https://github.com/dotnet/core-setup/issues/4365) via [**ZenHub**](https://www.zenhub.com/)
